build/index.html

```diff
@@ -821,6 +832,18 @@ <h5 class="head-5 text--white margin-bottom-medium">Update a customer</h5>
       </td>
       <td>The customer&rsquo;s full name (first and last).</td>
     </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td class="nowrap">
+        <strong class="is-block text--navy">plan</strong>
+        <span class="is-block text--transparent text-3">string</span>
+        <span class="text--transparent text-3">optional, default is &ldquo;standard&rdquo;</span>
+      </td>
+      <td>
+        The customer&rsquo;s plan determines content accessibility. Values can be one of the following strings: <code>free</code>, <code>standard</code>.
+        <br><br>
+        This should be set depending on your product type. Use <code>free</code> if you are using subscription that is free with registration. In any other case, you should use <code>standard</code>.
+      </td>
```

Comment:
This looks like a copy of the plan row, when it should be the password row.

Reply:
Updated this in newest commit. Thanks!